docs/examples/fd_fitting_basic/index.rst

```diff
@@ -0,0 +1,194 @@
+chain model, and then use those estimates as initial guesses to fit the twistable model.
+
+Let's set up the Odijk model and create the fit::
+
+    m_odijk = pylake.inverted_odijk('DNA').subtract_independent_offset() + pylake.force_offset('DNA')
+    fit_odijk = pylake.FdFit(m_odijk)
+
+Considering that this model only describes the lower part of the curve, we have to
+extract the data that is relevant to us. We can obtain this data from the ``FdCurve``
+as follows::
+
+    force = fd_curve.f.data
+    distance = fd_curve.d.data
+
+We only wish to use the forces below 30, so we filter the data according to this
+requirement::
+
+    mask = force < 30
+    distance = distance[mask]
+    force = force[mask]
+
+Now we are ready to add this data to the fit. Note that it is important to constrain the distance offset, as this
+provides a lot of additional freedom in the model::
+
+    fit_odijk.add_data("Inverted Odijk", force, distance)
+    fit_odijk["DNA/d_offset"].upper_bound = .01
+    fit_odijk["DNA/d_offset"].lower_bound = -.01
+
+And fit the model::
+
+    >>> fit_odijk.fit()
+
+    Fit
+      - Model: DNA(x-d)_with_DNA
+      - Equation:
+          f(d) = argmin[f](norm(DNA.Lc * (1 - (1/2)*sqrt(kT/(f*DNA.Lp)) + f/DNA.St)-(d - DNA.d_offset))) + DNA.f_offset
+
+      - Data sets:
+        - FitData(Inverted Odijk, N=959)
+
+      - Fitted parameters:
+        Name                  Value  Unit      Fitted      Lower bound    Upper bound
+        ------------  -------------  --------  --------  -------------  -------------
+        DNA/d_offset    -0.00601252  [au]      True              -0.01           0.01
+        DNA/Lp          44.2558      [nm]      True               0            100
+        DNA/Lc           2.80085     [micron]  True               0            inf
+        DNA/St        1714.56        [pN]      True               0            inf
+        kT               4.11        [pN*nm]   False              0              8
+        DNA/f_offset     0.0392458   [pN]      True              -0.1            0.1
+
+Set up the Twistable worm like chain model
+------------------------------------------
+
+Set up a twistable worm like chain model with a distance and force offset. By default,
+the `twistable_wlc` model provided with pylake is defined as distance as a function of
+force. Typically, we want to fit force as a function of distance however. To achieve
+this, we can invert the model using its `invert` function. However, this can be slow
+in certain cases, as it requires an inversion for each data point. Luckily we have a
+faster way of achieving this in pylake, which is to use the dedicated `inverted_twistable_wlc`
+model.
+
+Again, we incorporate an offset in both distance and force to compensate for small
+offsets that may exist in the data:::
+
+    m_dna = pylake.inverted_twistable_wlc('DNA').subtract_independent_offset() + pylake.force_offset('DNA')
```

Comment:
Again, make sure it fits without horizontal scrolling. Line breaks are fine.

Reply:
👍